### PR TITLE
feat(analytics): Track project creation

### PR DESCRIPTION
**Pull request by jeffkwoh**
Related: 
- https://github.com/getsentry/reload/pull/141
- getsentry/reload#142
- https://github.com/getsentry/getsentry/pull/3562

Part of: https://www.notion.so/sentry/Default-Alert-Options-450bbbc6f83b4ba29350fb9a59cf5746

### CODE REVIEW

**Comment by jeffkwoh** [comment]
@billyvg Are there any gotchas with adding logging/tracking to `componentDidMount`? I'm generally thinking about scenarios where react remounts the component but the user didn't semantically navigate away from the page.

For instance, if we switch orgs but semantically stay on the page. Although, I don't think this should be a concern because switch orgs would place you in the issue page.

File: src/sentry/static/sentry/app/views/projectInstall/createProject.jsx [hunk 1]
```diff
@@ -63,6 +66,19 @@ class CreateProject extends React.Component {
     }
   }
 
+  componentDidMount() {
+    super.componentDidMount();
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/projectInstall/createProject.jsx]: This isn't needed afaik

**Review by billyvg — APPROVED** [review]
`componentDidMount` is going to be the best place to put it. For views, it's likely they generally will only get unmounted/mounted due to navigation.

File: src/sentry/static/sentry/app/views/projectInstall/createProject.jsx (hunk 1 above)
Comment (jeffkwoh) [reply src/sentry/static/sentry/app/views/projectInstall/createProject.jsx]: Done.

File: src/sentry/static/sentry/app/views/projectInstall/createProject.jsx [hunk 2]
```diff
@@ -63,6 +64,22 @@ class CreateProject extends React.Component {
...
+    // TODO(jeff): Change key to AlertDefaultExperiment on the real experiment run
+    logExperiment({
+      organization: this.props.organization,
+      key: 'AlertDefaultExperimentTmp',
+      unitName: 'org_id',
+      unitId: parseInt(this.props.organization.id, 10),
+      param: 'exposed',
+    });
+    trackAnalyticsEvent({
+      eventKey: 'new_project.visit',
```
Comment (maheskett) [inline src/sentry/static/sentry/app/views/projectInstall/createProject.jsx]: We generally use `trackAdhocEvent` for higher volume events due to certain rate limits. Also, do you need to track this analytics event in addition to logging the experiment?

Comment (jeffkwoh) [reply src/sentry/static/sentry/app/views/projectInstall/createProject.jsx]: Addressed in: https://github.com/getsentry/sentry/pull/17256

**Pull request merged**